index.html

```diff
@@ -173,5 +263,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
+            <h2 class="why-choose-us-head">why choose us</h2>
```

Comment:
take in consideration that same title and subtitle styles are on all sections on the page. So you will have to invent new class names. 
Check out my message in general channel in slack